asllib/tests/ASLTypingReference.t/TypingRule.AnnotateCallActualsTyped.bad2.asl

```diff
@@ -0,0 +1,13 @@
+func xor_extend{N, M}(x: bits(N), y: bits(M)) => bits(N)
+begin
+    return x XOR ZeroExtend{N, M}(y);
+end;
+
+func main() => integer
+begin
+    var bv1 = Zeros{64};
+    var bv2 = Zeros{32};
+    - = xor_extend{64, 32}(bv1, bv2);
+    - = xor_extend{64}(bv1); // Illegal: missing argument for `y`.
```

Comment:
nit: perhaps it's worth having both parameters here, so that the missing argument is the only error? (and then reflecting in prose)